src/Microsoft.AspNetCore.Authentication.OAuth/OAuthChallengeProperties.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Authentication.OAuth
+{
+    public class OAuthChallengeProperties : AuthenticationProperties
+    {
+        /// <summary>
+        /// The property for the "scope" parameter value being used for a challenge request.
+        /// </summary>
+        public static readonly string ScopeParameterPropertiesKey = "scope";
```

Comment:
Nit: I think you can safely drop the Properties in all of these constants without any loss of clarity, i.e. ScopeParameterKey